composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "ricbra/php-discogs-api",
+    "name": "davalb/php-discogs-api",
```

Comment:
Simplified version of <code>composer.json</code> with only the relevant parts for this to work:
```json
{
    "repositories":
    [
            {
                "type": "vcs",
                "url": "https://github.com/davalb/php-discogs-api"
            }
    ],
    "require":
    {
        "ricbra/php-discogs-api": "dev-master"
    }
}
```

Note: you have to revert the commit with the renaming of the package name also on your branch/fork. So the composer.json should contain "ricbra/php-discogs-api".

For more information check out: https://getcomposer.org/doc/05-repositories.md#vcs

Could you remove your package from packagist so people don't get confused which one to use afterwards? 

If you need help with anything else give me a shout.